src/api/store/download.py

```diff
@@ -464,10 +497,11 @@ def _all_actions_download(self):
       data.append([community] \
         + self._get_action_info_cells(action))
 
-    for community in Community.objects.filter(is_deleted=False):
-      community_reported_rows = self._get_reported_data_rows(community)
-      for row in community_reported_rows:
-        data.append([community.name] + row)
+    # BHN - Removing state/partner reported from all actions download
```

Comment:
remove if not needed

Reply:
Leaving that comment in in case someone complains, it will remind me to add it back